amazon/evidently/README.md

```diff
@@ -0,0 +1,63 @@
+# Secrets Manager
```

Comment:
Wrong name! 😏

Reply:
Guess we both know which module I copy-pasted from.